cookbook/routing/index.rst

```diff
@@ -9,6 +9,8 @@ Routing
     redirect_in_config
     method_parameters
     service_container_parameters
-    custom_route_loader
     redirect_trailing_slash
     extra_information
+    extending_routing
+        custom_route_loader
+        multiple_routers
```

Comment:
This doesn't work in the table of contents. You will probably want to update `/cookbook/routing/map.rst.inc` instead. But I am not sure if the symfony.com theme is able to render that properly (the platform.sh preview will tell us).